javatests/arcs/core/entity/EntityDereferencerFactoryTest.kt

```diff
@@ -18,12 +19,24 @@ import arcs.core.storage.StorageKeyManager
 import arcs.core.storage.keys.ForeignStorageKey
 import arcs.core.storage.keys.RamDiskStorageKey
 import arcs.core.storage.testutil.DummyStorageKey
+import kotlinx.coroutines.test.runBlockingTest
 import org.junit.After
 import org.junit.Before
 import kotlin.test.assertFailsWith
 
 @RunWith(JUnit4::class)
 class EntityDereferencerFactoryTest {
+  val SCHEMA1 = Schema(
+    emptySet(),
+    SchemaFields(mapOf("name" to FieldType.Text), emptyMap()),
+    "abc"
+  )
+
+  val SCHEMA2 = Schema(
+    emptySet(),
+    SchemaFields(mapOf("foo" to FieldType.Number), emptyMap()),
+    "def"
+  )
```

Comment:
Can you put these in a companion object at the bottom of the class? That will make them proper static cosntants.